exercises/concept/castle-dinner/.docs/introduction.md

```diff
@@ -16,8 +16,8 @@ In turn this means that the compiler will automatically check for null reference
 ## Falsey value
 
 To be able to handle `nil` values, are there certain approaches that can be taken.
-`nil` is a [falsey][truhy-falsey] value as well are `false`.
-This means that in if statements and other places where a falesy or truthy value is expected it will be treated as false.
+`nil` is a [falsey][truthy-falsey] value as well are `false`.
+This means that in if statements and other places where a falsey or truthy value is expected so will it be treated the same as `false`.
```

Comment:
```suggestion
This means that in if statements and other places where a falsey or truthy value is expected it will be treated the same as `false`.
```